docs/05-technical-reference/00-custom-resources/_sidebar.md

```diff
@@ -1,7 +1,6 @@
 * [Home](/)
 * [Technical Reference](../README.md)
   * [Custom resources](README.md)
-    * [Application](ac-01-application.md)
```

Comment:
The Application Custom Resource has not been removed from Kyma repo (yet!!!) and is installed as part of compass-runtime-agent until it will be finally migrated to Application Connector Module. We should not remove its description.